feature/platform/transceiver/tests/zr_pm_test/metadata.textproto

```diff
@@ -0,0 +1,20 @@
+# proto-file: third_party/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+uuid: "f5d5c225-8900-42fd-8771-8614895ae036"
+plan_id: "TRANSCEIVER-6"
+description: "Telemetry: 400ZR Optics PM (Q, Pre-FEC BER, ESNR) streaming"
+testbed: TESTBED_DUT_400ZR
+platform_exceptions: {
+  platform: {
+    vendor: ARISTA
+  }
+  deviations: {
+    connect_retry: true
```

Comment:
Are all these deviations being used in this test? If not, we shouldn't be adding them here. 

The metadata.textproto for each test should only have the deviations which are being used by the test . 

I see the function `[ConfigureDefaultNetworkInstance](https://source.corp.google.com/piper///depot/google3/third_party/openconfig/featureprofiles/internal/fptest/networkinstance.go;bpv=1;bpt=1;rcl=613934699;l=31?gsn=ConfigureDefaultNetworkInstance&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fgoogle3%3Flang%3Dgo%3Fpath%3Dthird_party%2Fopenconfig%2Ffeatureprofiles%2Finternal%2Ffptest%2Ffptest%23func%2520ConfigureDefaultNetworkInstance)` uses default_network_instance. But couldnt find the others being used.

Reply:
Done.